scripts/modal.js

```diff
@@ -0,0 +1,82 @@
+const body = document.querySelector('body');
+// const featuredModal = document.querySelector('.featuredmodal');
+const featuredbtn = document.getElementById('muiltipostbtn');
+const projectBtn = document.querySelectorAll('.openProject button');
+
+const popUp = document.createElement('div');
+popUp.classList.add('popup');
+
+const projectList = [
+  {
+    id: 'proj1',
+    title: 'Multi Post Stories',
+    closebtn: '<i class="fa fa-times" aria-hidden="true"></i>',
+    html: 'html',
+    bootstrap: 'bootstrap',
+    ruby: 'Ruby On Rails',
+    projectImage: './images/mobilemodal.png',
+    projectDesc: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam amet animi voluptate doloribus at quaerat, vitae dicta sunt possimus voluptatum, facilis vel? Eligendi, harum ipsum excepturi sint at quasi? Repellendus eveniet suscipit quibusdam inventore architecto minus eligendi adipisci. Harum atque voluptate dolore fugiat pariatur eius ullam architecto porro aspernatur nihil.',
+    projectbtnlive: 'https://nemwel-boniface.github.io/personal-portfolio/',
+    projectbtnsource: 'https://nemwel-boniface.github.io/personal-portfolio/',
+  },
+  {
+    id: 'proj6',
+    title: 'Multi Post Stories',
+    closebtn: '<i class="fa fa-times" aria-hidden="true"></i>',
+    html: 'html',
+    bootstrap: 'bootstrap',
+    ruby: 'Ruby On Rails',
+    projectImage: './images/mobilemodal.png',
+    projectDesc: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam amet animi voluptate doloribus at quaerat, vitae dicta sunt possimus voluptatum, facilis vel? Eligendi, harum ipsum excepturi sint at quasi? Repellendus eveniet suscipit quibusdam inventore architecto minus eligendi adipisci. Harum atque voluptate dolore fugiat pariatur eius ullam architecto porro aspernatur nihil.',
+    projectbtnlive: 'https://nemwel-boniface.github.io/personal-portfolio/',
+    projectbtnsource: 'https://nemwel-boniface.github.io/personal-portfolio/',
+  },
+];
+
```

Comment:
- [ ] Nice job 👏  on storing your projects in an array of objects however I can only see two projects' information. All of your 6 projects should be using information or data stored in your array of objects(const projectList=[{},{},{}]),

Reply:
Hello @andersonlebon  thank you for this review.
- I was at first looping through all the buttons to select the details of the modal popup from the array's object.
- I have however made the required changes